lua/nvim-treesitter/parsers.lua

```diff
@@ -646,6 +646,7 @@ list.gaptst = {
   install_info = {
     url = "https://github.com/gap-system/tree-sitter-gaptst",
     files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
```

Comment:
```suggestion
```
Only needed if that's not the default branch. What is missing is `lockfile.json` entries.

Reply:
Ah that makes sense, thanks! I've removed the branch and updated the lockfile, hopefully CI passes now.

Reply:
Looks good, yes! (Sorry, didn't catch this earlier.)